### PR TITLE
Add 🛠 topic emoji for users using beta versions

### PR DESCRIPTION
**Pull request by Soheab**
Aka installed from the repo 

Not tested or anything, was done on mobile via GitHub but that doesn't matter here.

### CODE REVIEW

**Review by ooliver1 — APPROVED** [review]

**Review by koala9712 — APPROVED** [review]
🛠️

**Comment by TAG-Epic** [comment]
Is this a common occurrence?

**Comment by Soheab** [comment]
> Is this a common occurrence?

Kinda, yes.

File: cogs/help.py [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "🤚": "Help Needed",
     "🔥": "Active",
     "⏳": "Stalled",
+    "🛠️": "Development",
```
Comment (Skelmis) [inline cogs/help.py:49]: I feel this should be `Development Version` as just `Development` could be misleading

**Review by Skelmis — APPROVED** [review]

**Pull request merged**